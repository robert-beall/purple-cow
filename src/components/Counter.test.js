import renderer from 'react-test-renderer';
import Counter from './Counter.js';

it('render counter without button', async () => {
  let testFunction = () => console.log("test");

  const component = renderer.create(
    <Counter updateButton={false}
             message="Purple Cows have joined the team!"/>,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
