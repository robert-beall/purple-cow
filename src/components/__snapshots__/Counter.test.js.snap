// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render counter without button 1`] = `
<div>
  <p>
     
    Purple Cows have joined the team!
  </p>
</div>
`;
